### DISCUSSION TITLE
Mutiple email support

### DISCUSSION THREAD

**Discussion by nexmoe** [appwrite/appwrite#5759] [Ideas]
When a user discard his email, he can use other emails he set to log in. 
See GitHub's email system for a similar example.

**Comment by joeyouss** [comment]
Hi
I am sorry but is it related to Discord? Can you please elaborate?

**Comment by koodeau** [comment]
Is it about Email authentication or related to GitHub OAuth which could mask your email address from public?

**Comment by nexmoe** [comment]
Similar to https://github.com/settings/emails
A user can bind multiple emails, so when one email is not available, they can use another email for login and other operations

**Comment by byawitz** [comment]
For now, the user must be identified either by email or by phone.
Meaning in case the user hide is email - like Github, Apple, etc. - then another user will be created.

Check [this](https://discord.com/channels/564160730845151244/1126169242371371048/1126171829900431453) Discord thread
